src/app/components/core/developer/additional-services/additional-service-modal/additional-service-modal.component.ts

```diff
@@ -459,7 +462,8 @@ export class AdditionalServiceModalComponent implements OnInit, OnDestroy {
           } else {
             // if service.status !== status then
             // throw an error to re-fetch:
-            if (service.status !== 'running') {
+            let expectedStatus = status ? 'running' : 'shutdown';
```

Comment:
I'm not clear on this status ? 'running' : 'shutdown';
Cannot the service be in failed, unresponsive state?

Reply:
This is the expected status after enabling or disabling the service. Expected service state is `running` when we enable the service and `shutdown` when we disable it.